manifests/base/application-controller-deployment/kustomization.yaml

```diff
@@ -2,5 +2,8 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
+- ../application-controller-roles
 - argocd-application-controller-service.yaml
+- argocd-application-controller-statefulset.yaml
```

Comment:
Just confirming: should the base/deployment also include the Statefulset scaled to zero or it should just remove this resource?

Reply:
Kept it as a patch so that we do not try to delete something that is not present. Now that we have moved roles into a separate directory, we do not need to apply statefulsets before deployment.